Platform.PCL/Realm.PCL/Extensions/RealmResultsCollectionChangedPCL.cs

```diff
@@ -22,6 +22,9 @@
 
 namespace Realms
 {
+    /// <summary>
+    /// Realm results collection changed.
+    /// </summary>
```

Comment:
This header is not helpful. If it's there just to appease StyleCope, better put a suppression attribute instead.